### PR TITLE
Tables: make `toCountTable` actually count the elements of the input openArray.

### PR DESCRIPTION
**Pull request by moigagoo**

### CODE REVIEW

File: lib/pure/collections/tables.nim [hunk 1]
```diff
@@ -968,7 +968,7 @@ proc initCountTable*[A](initialSize=64): CountTable[A] =
 proc toCountTable*[A](keys: openArray[A]): CountTable[A] =
   ## creates a new count table with every key in `keys` having a count of 1.
```
Comment (GULPF) [inline lib/pure/collections/tables.nim]: Update the doc comment as well

Comment (moigagoo) [reply lib/pure/collections/tables.nim]: Thanks! Fixed.

**Comment by cooldome** [comment]
A bit of discussion here: https://forum.nim-lang.org/t/3432

**Comment by Araq** [comment]
This needs to be mentioned in the changelog but is an excellent improvement.

**Comment by moigagoo** [comment]
Added the changelog entry: https://github.com/nim-lang/Nim/pull/6977/commits/a8b0a8a92da03593c3fe52dfbca8ee4eeed286d3

**Comment by dom96** [comment]
Nice, looks good.

**Pull request merged**